### PR TITLE
Amazon Linux 2023 (for Uyuni)

### PR DESCRIPTION
**Pull request by juliogonzalez**
# Description

Amazon Linux 2023

Only for Uyuni for now.

# Target branches

* Which documentation version does this PR apply to? **Uyuni**
* Does this content need to be backported? **Not for now, at least.**
* Whenever possible, cross-reference each backport PR here, so that all backports can be easily accessed from the description. **None**

- [x] master
- [ ] manager-4.3

# Links
- This PR tracks issue: https://github.com/SUSE/spacewalk/issues/20799
- Related development PR: https://github.com/uyuni-project/uyuni/pull/7829

### CODE REVIEW

File: modules/client-configuration/pages/clients-amazon.adoc [hunk 1]
```diff
@@ -84,11 +84,21 @@ The channels you need for this procedure are:
 | amazonlinux2-core
 | amazonlinux2-uyuni-client
 
+| {amazon} 2023
+| amazonlinux2023
```
Comment (juliogonzalez) [inline modules/client-configuration/pages/clients-amazon.adoc:88]: Strictly speaking, this is not called `core` anymore, unlike on Amazon Linux 2, but I guess we can keep the title for the table as it is?

File: .changelog [hunk 2]
```diff
@@ -4,6 +4,7 @@
 # - Fixed error in Bat section of Upgrade Guide (bsc#1234567)
 # For guidelines: https://en.opensuse.org/openSUSE:Creating_a_changes_file_(RPM)
 
+- Document Amazon Linux 2023
```
Comment (keichwa) [inline .changelog:7]:
```suggestion
- Documented Amazon Linux 2023 as supported by Uyuni in Client
  Configuration Guide
```

File: modules/client-configuration/pages/clients-amazon.adoc [hunk 3]
```diff
@@ -84,11 +84,21 @@ The channels you need for this procedure are:
...
+| amazonlinux2023-uyuni-client
+
 |===
 
+
+[NOTE]
+====
+For {amazon} 2, make sure you also add and sync `amazonlinux2-extra-docker` channel if you plan to use Docker at your {amazon} instances.
```
Comment (keichwa) [inline modules/client-configuration/pages/clients-amazon.adoc:96]:
```suggestion
For {amazon} 2, make sure you also add and synchronize `amazonlinux2-extra-docker` channel if you plan to use Docker at your {amazon} instances.
```

File: modules/client-configuration/pages/clients-amazon.adoc [hunk 4]
```diff
@@ -84,11 +84,21 @@ The channels you need for this procedure are:
...
+====
+
 [NOTE]
 ====
-Make sure you also add and sync `amazonlinux2-extra-docker` channel if you plan to use Docker at your {amazon} instances.
+For {amazon} 2023, make sure you also add and sync `amazonlinux2023-kernel-livepatch` channel if you plan to use Kernel Live patches at your {amazon} instances.
```
Comment (keichwa) [inline modules/client-configuration/pages/clients-amazon.adoc:101]:
```suggestion
For {amazon} 2023, make sure you also add and synchronize `amazonlinux2023-kernel-livepatch` channel if you plan to use Kernel Live patches at your {amazon} instances.
```

**Review by keichwa — APPROVED** [review]
Just cosmetic proposals.

File: modules/client-configuration/pages/clients-amazon.adoc (hunk 1 above)
Comment (keichwa) [reply modules/client-configuration/pages/clients-amazon.adoc:88]: ok, thanks!

**Review by keichwa — APPROVED** [review]

File: modules/client-configuration/pages/clients-amazon.adoc (hunk 1 above)
Comment (juliogonzalez) [reply modules/client-configuration/pages/clients-amazon.adoc:91]: Done.

**Review by 0rnela — APPROVED** [review]

**Review by jcayouette — APPROVED** [review]

**Review by keichwa — APPROVED** [review]

**Comment by juliogonzalez** [comment]
https://github.com/uyuni-project/uyuni/pull/7829 (code) got merged, so merging the doc as well.

**Pull request merged**